larray_editor/api.py

```diff
@@ -339,12 +339,13 @@ def restore_display_hook():
                           [ '', 1664780726569649730, -9196963249083393206, -7664327348053294350]])
 
     # test autoresizing
-    arr8 = la.zeros('a=a_long_label,another_long_label')
+    arr8 = la.zeros('a=a_long_label,another_long_label; b=this_is_a_label,this_is_another_one')
```

Comment:
it would be nice to give meaningful names to those arrays: either what do they test, or what is their "special" characteristic, eg:
```python
>>> test_column_resize1 = la.zeros('a=a_long_label,another_long_label; b=this_is_a_label,this_is_another_one')
>>> # or
>>> long_labels = la.zeros('a=a_long_label,another_long_label; b=this_is_a_label,this_is_another_one')
```